neqo-transport/src/ecn.rs

```diff
@@ -138,6 +152,28 @@ impl EcnInfo {
             && (self.baseline - prev_baseline)[IpTosEcn::Ce] > 0
     }
 
+    pub fn on_packets_lost(&mut self, lost_packets: &[SentPacket]) {
+        if let EcnValidationState::Testing {
+            probes_sent,
+            probes_lost,
+        } = &mut self.state
+        {
+            *probes_lost += lost_packets
+                .iter()
+                .filter(|p| p.packet_type() == PacketType::Initial && p.ecn_mark().is_ecn_marked())
+                .count();
+            // If we have lost all initial probes a bunch of times, we can conclude that the path
+            // is not ECN capable.
```

Comment:
```suggestion
            // If we have lost all initial probes a bunch of times, we can conclude that the path
            // is not ECN capable and likely drops all ECN marked packets.
```

The path is not only not capable, but also likely drops all packets in this case, right? If so, differentiating the two might be easier for folks to comprehend.